### PR TITLE
Updated spacing and sizing on indicator menus

### PR DESCRIPTION
**Pull request by mouakiang**

### CODE REVIEW

File: .vscode/settings.json
Comment (dizzyjaguar) [inline .vscode/settings.json:1]: lets leave `.vscode/settings.json` out of the github repo. If you can add that too the `.gitignore` and then remove this from the commit

Comment (mouakiang) [reply .vscode/settings.json:1]: okay!

File: src/routes/player.tsx
Comment (dizzyjaguar) [inline src/routes/player.tsx:1]: Looks like you have `justify-evenly` and `justify-around` on the same default media query, it should just be one

**Review by dizzyjaguar — APPROVED** [review]
good work 👍🏼

**Pull request merged**